src/sentry/static/sentry/app/views/organizationIntegrations/installedIntegration.jsx

```diff
@@ -77,16 +79,29 @@ export default class InstalledIntegration extends React.Component {
           <Box px={2} flex={1} style={style}>
             <IntegrationItem integration={integration} />
           </Box>
-          {this.hasConfiguration() && (
-            <Box mr={1}>
-              <Button
-                size="small"
-                to={`/settings/${orgId}/integrations/${provider.key}/${integration.id}/`}
-              >
-                {t('Configure')}
-              </Button>
-            </Box>
-          )}
+          {integration.status === 'active' &&
+            this.hasConfiguration() && (
+              <Box mr={1}>
+                <Button
+                  size="small"
+                  to={`/settings/${orgId}/integrations/${provider.key}/${integration.id}/`}
+                >
+                  {t('Configure')}
+                </Button>
+              </Box>
+            )}
+          {integration.status === 'disabled' &&
+            this.hasConfiguration() && (
```

Comment:
This doesn't seem right, the reinstall button would only show up if it's configurable because of the `hasConfiguration`?